### PR TITLE
update readme with correct kibana action

### PR DESCRIPTION
**Pull request by kwmonroe**
- `deploy-dashboard` should be `load-dashboard`
- adjust formatting on `http://<kibana-public-ip>/dashboards` so the `<` and `>` don't get rendered out

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,11 +42,11 @@ services. Load the example dashboard and you're ready to go!
 With our services on the way, once the model has settled deploy the beats
 dashboard in Kibana.
 
-    juju action do kibana/0 deploy-dashboard dashboard=beats
+    juju action do kibana/0 load-dashboard dashboard=beats
 
 You will notice new indexes in kibana, and some default demonstration dashboards
 have been loaded. This is a full beats suite dashboard. You can now navigate
-to http://<kibana-public-ip>/dashboards and load up the topbeat dash and you're
+to `http://<kibana-public-ip>/dashboards` and load up the topbeat dash and you're
```
Comment (lazypower) [inline README.md:49]: Perfect with the ticks. thanks!

File: README.md [hunk 2]
```diff
@@ -42,11 +42,11 @@ services. Load the example dashboard and you're ready to go!
 With our services on the way, once the model has settled deploy the beats
 dashboard in Kibana.
 
-    juju action do kibana/0 deploy-dashboard dashboard=beats
+    juju action do kibana/0 load-dashboard dashboard=beats
```
Comment (lazypower) [inline README.md:45]: Nicely done here. sorry for the stale info.

**Review by lazypower — APPROVED** [review]
Perfect +1 LGTM

**Comment by mbruzek** [comment]
+1

**Pull request merged**